flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/table/RowDataDebeziumDeserializeSchema.java

```diff
@@ -480,6 +517,39 @@ public Object convert(Object dbzObj, Schema schema) {
         };
     }
 
+    private DeserializationRuntimeConverter createArrayConverter(ArrayType arrayType) {
+        final Class<?> elementClass =
+                LogicalTypeUtils.toInternalConversionClass(arrayType.getElementType());
+        final DeserializationRuntimeConverter elementConverter =
+                createConverter(arrayType.getElementType());
+        return (dbzObj, schema) -> {
+            if (dbzObj instanceof List) {
+                List<?> elements = (List<?>) dbzObj;
+                final Object[] elementArray =
+                        (Object[]) Array.newInstance(elementClass, elements.size());
+                for (int i = 0; i < elements.size(); i++) {
+                    elementArray[i] = elementConverter.convert(elements.get(i), schema);
+                }
+                return new GenericArrayData(elementArray);
+            } else if (dbzObj instanceof String) {
+                // for SET datatype in mysql, debezium will always return a string split by comma
```

Comment:
Does debezium object contain any type information that can use to infer the object data is SET?